.github/workflows/packages-release.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow is intended to be run manually when cutting an official release.
+# The workflow calls the reusable packages workflow from packages.yaml and then
+# generates an SBOM (Software Bill of Materials).
+name: 'packages-release'
+
+on:
+  workflow_dispatch:
+    branches: [release/1.2]
+
+jobs:
+  packages:
+    # TODO: Replace sbom-generation branch w/ release/1.2
+    uses: nlcamp/iot-identity-service/.github/workflows/packages.yaml@sbom-generation-1.2
+  sbom:
+    needs: packages
+    runs-on: ubuntu-latest
```

Comment:
Pin a specific version like the other workflows. Otherwise a new OS release can cause breakage.